mne/channels/tests/test_montage.py

```diff
@@ -240,18 +240,23 @@ def test_documented():
     pytest.param(
         partial(read_custom_montage, head_size=None, unit='n/a'),
         ('346\n'  # XXX: this should actually race an error 346 != 4
-         'EEG\t      F3\t -62.027\t -50.053\t      85\n'
-         'EEG\t      Fz\t  45.608\t      90\t      85\n'
-         'EEG\t      F4\t   62.01\t  50.103\t      85\n'
-         'EEG\t      FCz\t   68.01\t  58.103\t      85\n'),
+         'FID\t      LPA\t -120.03\t      0\t      85\n'
+         'FID\t      RPA\t  120.03\t      0\t      85\n'
+         'FID\t      Nz\t   114.03\t     90\t      85\n'
```

Comment:
I see that you have this first column that tells you the channel type. Do we ignore it so far because we only had EEG? do you have a public file that contains this FID?

Reply:
Yes the column is currently ignored. I was thinking about making use of it too, but then wondered if it would even make sense since we‘d still have to identify the fiducials via their respective names anyway. I do have an elp file here but don’t remember where I got it from... probably EEGLAB? Can share later today

Reply:
well we could enforce something like if it is of type FID it must be called LPA, RPA or Nz and if we have one we must have the three but maybe it's an over kill.

I am ok to merge as it is. What do you think?